driver/makedepend.ml

```diff
@@ -591,6 +593,8 @@ let main () =
         " Dump the delayed dependency map for each map file";
      "-I", Arg.String (add_to_list Clflags.include_dirs),
         "<dir>  Add <dir> to the list of include directories";
+     "-nocwd", Arg.Set nocwd,
+        "Do not add current working directory to list of include directories";
```

Comment:
Why are you eliding the `the` in `add to list of include directories`? It doesn't seem to be a Frenchism?

Reply:
It was goinng past 80 columns :P

Reply:
Then you just need to split the text with
```ocaml
"-nocwd", Arg.Set nocwd,
  "Do not add current working directory to \
    the list of include directories";
```